themes/base.theme.bash

```diff
@@ -85,37 +85,45 @@ RBENV_THEME_PROMPT_SUFFIX='|'
 RBFU_THEME_PROMPT_PREFIX=' |'
 RBFU_THEME_PROMPT_SUFFIX='|'
 
-GIT_EXE=$(which git 2> /dev/null || true)
-P4_EXE=$(which p4 2> /dev/null || true)
-HG_EXE=$(which hg 2> /dev/null || true)
-SVN_EXE=$(which svn 2> /dev/null || true)
-
-# Check for broken SVN exe that is caused by some versions of Xcode.
-# See https://github.com/Bash-it/bash-it/issues/1612 for more details.
-if [[ -x "$SVN_EXE" ]]; then
-	if ! "$SVN_EXE" --version > /dev/null 2>&1; then
-		# Unset the SVN exe variable so that SVN commands are avoided.
-		SVN_EXE=""
+: "${GIT_EXE:=$SCM_GIT}"
```

Comment:
Are those lines needed now?
The init function you created should take care of those variables now

Reply:
I would like to keep them, if that's OK? 😅

Reply:
why though? I am curious :smile:

Reply:
My brain worms like variables that are referenced elsewhere to be initialized unconditionally. 

This alsö helps with static analyzers like `shfmt` and `shellcheck` such that they can see that the variables are in fact defined without having to evaluate the function call. Of course, a static analyzer wouldn't be very useful if it couldn't follow such a simple function so this isn't super likely to be flagged for anything, but my brain worms like consistency even when the "other place" is literally the next line and the function is simple. 

I'm very good at code cleanup because I've learned to let my brain worms help and to appease them when they're a little more pedantic than is practically useful. 🤣🙃 (Something not "cleanup" takes me much longer, like the defaults completions! 😳)

Reply:
Alright- your brain worms make sense :smile: